WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ border-bottom:1px solid @SeparatorColor@; @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = vertical-align:bottom;
+TaskReadingPaneRight-ListViewItemDate                       = align:right; padding-top:16px;
```

Comment:
I think align would be an invalid css property. Its mostly available as attribute in table and related tags.